src/data/pages/interpreting-hubble-plot-1.json

```diff
@@ -19,6 +19,7 @@
     "sources": [
```

Comment:
We're going to need this same data (or at least kind of data) across a bunch of different pages, and in fact, probably across the expanding universe investigation and the supernova investigation, so I think I liked your idea about pulling in this data dynamically through axios.  I'm going to do so with the hubble plot so if you'd like, go ahead and make a separate ticket for refactoring your work to use axios, and you can take a look at what I have when I've pushed it up, and keep on keeping on with what you have so far.

Reply:
Do I not assign the ticket to this sprint. I'm 100% positive that the refactor you're speaking about is going to take a few days.

Reply:
I don't think it's as big a lift as you fear.  As I said, I'm already using axios for the data powering my hubble plot and seems fine.  After you finish the next chunk of work that you would like to add to this pull request we can sit down together and walk through it.  At that point it'll probably be time to merge this branch into master as well.  I don't want our work to stray too far apart since the next next step is going to be integrating your selector and my hubble plot.

Reply:
Reviewing what you've done in your axios example, I dont feel that another ticket would be necessary. I've already adjust the code to do this.